molecule/molecule-testing/src/commonTest/java/app/cash/molecule/testing/MoleculeTestingTest.kt

```diff
@@ -258,4 +259,24 @@ class MoleculeTestingTest {
     assertSame(validateException, thrown)
     assertSame(moleculeException, thrown.suppressed.single())
   }
+
+  @Test
+  fun expectNoEvents() {
+    testMolecule({}) {
+      awaitItem()
+      expectNoEvents()
+    }
+  }
+
+  @Test
+  fun expectNoEventsFail() {
+    testMolecule({}) {
+      try {
+        expectNoEvents()
+        fail()
+      } catch (t: AssertionError) {
+        assertEquals(t.message, "Expected no events but found Item(kotlin.Unit)")
+      }
```

Comment:
Please use `assertFailsWith` rather than try/fail/catch. Right now you're getting lucky in that the `AssertionError` that would be thrown by `fail()` is being caught but is failing in the message check. Would rather using something a bit safer.

Reply:
Done!